kod/object/active/flag.kod

```diff
@@ -695,8 +695,8 @@ messages:
       return poClaimer;
    }
 
-   SetFaction(faction=FACTION_NEUTRAL)
-   "for admin use"
+   SetFaction(faction=FACTION_NEUTRAL,processClaim=FALSE)
+   "Updates the flag's faction. By default, this updates the faction only, which is needed when the territory game is recreated. Pass `processClaim=TRUE` to notify the territory game of the change."
```

Comment:
I mean, I literally don't know how the territory game works, so try to explain it to me like I'm 5.  If I am writing new code and I want to set a flag's faction, all of a sudden I see this mysterious extra parameter and a reference to recreating the territory game, which I thought I didn't care about.  I am instantly confused as to what value I should pass.  A perfect comment would make it obvious to me what I should do.

Words like "process" are like "manage" --- they convey almost no information.